app/file-upload/server/lib/FileUpload.js

```diff
@@ -389,6 +392,29 @@ export const FileUpload = {
 
 		request.get(fileUrl, (fileRes) => fileRes.pipe(res));
 	},
+
+	addJWTToFileUrlIfNecessary(message) {
```

Comment:
IMHO, we should check the `file` field outside this method. I mean, to me it looks weird returning a `message` from a method that deals with `JWT` stuff.

So, then the method will return the `JWT` itself, not a message object.